charts/super-agent-deployment/templates/configmap-subagent-configs.yaml

```diff
@@ -1,7 +1,6 @@
 {{- range $subAgentName, $subAgentConfig := .Values.config.subAgents -}}
 {{- $releasePrefix := include "newrelic.common.naming.fullname" $ -}}
-{{- $configName := printf "%s-%s" "subagent" $subAgentName -}}
```

Comment:
are you removing the "subagent-" just for the sake of simplicity or is there an other reason?

Reply:
yeap just to have a shorter name